modules/expectations.lua

```diff
@@ -95,6 +96,11 @@ function Expectations.Expectation(name, connection)
 
   --- Perform base validation of expectation and set result into `Test`
   function mt.__index:validate()
+    if self.isAtLeastOneFail == true then
+      if config.allFailedValidations == false or self.occurences == self.timesGE then
```

Comment:
@dboltovskyi Please check next use case:
 - config.allFailedValidations = true
 - one of ValidIf is failed
 - Upper bound of occurrences is not defined (`expectation:Times(AtLeast(1))`)
Will test step be failed in this case?

Reply:
@aderiabin Done in d5e8824